README.md

```diff
@@ -108,7 +108,7 @@ And then route this controller in your `urls.py`:
 
 ```python
 >>> from django.urls import include, path
->>> from django_modern_rest import Router
+>>> from django_modern_rest.routing import Router
```

Comment:
Can you please explore, why this example is not runned in CI? We must check the readme with `doctest`